backup_github.py

```diff
@@ -69,14 +73,17 @@ def backup(user, dest):
                 pass
 
 if __name__=="__main__":
-    parser = argparse.ArgumentParser(description="Clones all the public " \
-            "repositories from a GitHub account")
+    parser = argparse.ArgumentParser(description="Clones all the " \
+            "repositories from a GitHub account. " \
+            "Restricted to public ones if no password is given")
     parser.add_argument("user", type=str, help="GitHub username")
+    parser.add_argument("password", nargs='?', type=str, help="GitHub password")
```

Comment:
I could solve it via OAuth, but it's such a pain that I'd rather avoid it unless necessary.